Updated LandingPage->PageBuilder migration documentation (#984)
ezsystems/developer-documentation: docs/updating/updating_ez_platform.md

In the following instructions, replace `<version>` with the version of eZ Platform you are updating to (for example: `v2.5.0`).
If you are testing a release candidate, use the latest `-rc` tag (for example: `v2.5.0-rc1`).

> You may upgrade from any eZ Platform version to the latest 2.5 version. If you for instance are on 2.0. you may then

Reviewer: ```suggestion
You may upgrade from any eZ Platform version to the latest 2.5 version. For example, if you are using v2.0. you can then
```

> upgrade your distribution directly to latest 2.5, but follow all the database updates steps from 2.0 up to 2.5. It is

Reviewer: ```suggestion
upgrade your distribution directly to the latest 2.5 version, but follow all the database updates steps from 2.0 up to 2.5. It is
```

> recommended that you first run all the db schema updates up to 2.5 before you run other update scripts or procedures.

Reviewer: ```suggestion
recommended that you first run all the database schema updates up to 2.5 before you run other update scripts or procedures.
```
